include/stdlib.h

```diff
@@ -44,6 +48,10 @@ __Z_EXPORT char *realpath(const char * __restrict__, char * __restrict__) asm("_
  */
 __Z_EXPORT int mkstemp(char*) asm("__mkstemp_ascii");
 
+/**
+ * Use __getenv (@@A00423) which copies pointer to a buffer and is retained even after the environment changes
+ */
+__Z_EXPORT char* getenv(const char*) asm("@@A00423");
```

Comment:
Should this be__getenv instead of getenv since the commit is to use __getenv?

Reply:
This replaces all references of getenv with the ascii definition of __getenv which is pragma mapped to symbol @@A00423. I updated the comment to make it a bit more clear